Flepper.Tests.Unit/QueryBuilder/Commands/WhereFilterTests.cs

```diff
@@ -238,5 +238,64 @@ public void ShoulContainWhereWithNotEqualToNull()
                 .Should()
                 .Contain("WHERE [field] IS NOT NULL");
         }
+
+        [Fact]
+        public void ShoulContainLikeContains()
+        {
+            FlepperQueryBuilder.Select()
+                .From("table")
+                .Where("field").Contains("abc")
+                .Build()
+                .Trim()
+                .Should()
+                .Contain("LIKE");
+        }
+
+        [Fact]
+        public void ShoulContainLikeStartsWith()
+        {
+            FlepperQueryBuilder.Select()
+                .From("table")
+                .Where("field").StartsWith("abc")
+                .Build()
+                .Trim()
+                .Should()
+                .Contain("LIKE");
+        }
+
+        [Fact]
+        public void ShoulContainLikeEndsWith()
+        {
+            FlepperQueryBuilder.Select()
+                .From("table")
+                .Where("field").EndsWith("abc")
+                .Build()
+                .Trim()
+                .Should()
+                .Contain("LIKE");
```

Comment:
Test the full SQL Query and also check parameters

Reply:
Ok

Reply:
You still must test the FULL SQL Query, do not use Contains